ext_mongodb.php

```diff
@@ -227,6 +333,57 @@ public function __wakeup() : void;
 
 	<<__Native>>
 	public function selectServer(ReadPreference $readPreference): Server;
+
+/* {{{ APM */
+	public function addSubscriber( Monitoring\Subscriber $subscriber ) : void
+	{
+		$this->subscribers[] = $subscriber;
```

Comment:
Note: if you change subscribers to be a set, we should have a test to verify that a Subscriber added multiple times is only dispatched once.